test/androidTest/java/org/thoughtcrime/securesms/espresso/ConversationHelper.java

```diff
@@ -0,0 +1,93 @@
+package org.thoughtcrime.securesms.espresso;
+
+import android.support.test.InstrumentationRegistry;
+
+import org.thoughtcrime.securesms.R;
```

Comment:
This not placed in the correct order. Please refer to the coding style where that specify the imports order

Reply:
The coding guidelines are different than Android Studio?

I did none of this manually.